deployments/report-server-bucket-storage.nix

```diff
@@ -10,7 +10,7 @@
       # mount unsets PATH when running this
       device = "/run/current-system/sw/bin/s3fs#${resources.s3Buckets.report-server-logs.name}";
       fsType = "fuse";
-      options = [ "_netdev" "allow_other" "iam_role" ];
+      options = [ "_netdev" "allow_other" "nonempty" ("iam_role=${resources.iamRoles.report-role.name}") ];
```

Comment:
Do we really need nonemtpy? I would imagine the folder should be empty at the mount time. Probably report-server itself should depend on this mount to finish first. 

cc @cleverca22

Reply:
@domenkozar, it's more reliable, I would say -- should anything land there for any reason, the mount will still succeed.

But I'm also agnostic -- whatever you like.